runtime/port/unix/j9sysinfo.c

```diff
@@ -209,6 +210,43 @@ typedef struct wpar_info_format_t {
 #include "omrportptb.h"
 #include "ut_j9prt.h"
 
+typedef struct J9CgroupSubsystemMetricMap {
+	char *metricFilename;
+	char *metricTag;
+	char *metricKeyInFile;
+	char *metricUnit;
+	BOOLEAN isValueToBeChecked;
+} J9CgroupSubsystemMetricMap;
+
+static struct J9CgroupSubsystemMetricMap j9CgroupMemoryMetricMap[] = {
```

Comment:
These maps should be `const`.

Reply:
Thank you @keithc-ca for the review. Will be making changes accordingly.